jvb/pom.xml

```diff
@@ -285,12 +286,26 @@
       <version>5.5.2</version>
       <scope>test</scope>
     </dependency>
-    <dependency>
-      <groupId>io.kotlintest</groupId>
-      <artifactId>kotlintest-runner-junit5</artifactId>
-      <version>3.4.2</version>
-      <scope>test</scope>
-    </dependency>
+	<dependency>
+		<groupId>io.kotest</groupId>
```

Comment:
The rest of the file uses 2 spaces for indentation

Reply:
Tab snuck in there.